src/content/en/updates/2017/02/lighthouse-jan.md

```diff
@@ -0,0 +1,150 @@
+Chrome that may interfere with measuring performance, not the least of which
+are other extensions.
+
+In an attempt to make Lighthouse more consistent everywhere it's used, the
+extension's settings now has an option to disabled other Chrome extensions
+before running Lighthouse. If we have problems auditing your page, try enabling
+this option to see if it fixes the issue.
+
+![Disable other extensions setting](../../images/2017/02/lighthouse/image_7.png)
+
+## Lighthouse Viewer
+
+In HTML reports, you'll notice a new button with options for exporting the
+report in different formats. One of those options is "Open in Viewer". Clicking
+this button will send the report to the online [Viewer]([https://googlechrome.github.io/lighthouse/viewer/](https://googlechrome.github.io/lighthouse/viewer/)),
+where you can further share the report with users using Github.
+
+![Open in Viewer button](../../images/2017/02/lighthouse/image_8.png)
+
+![Open in Viewer result](../../images/2017/02/lighthouse/image_9.png)
+
+Behind the scenes, Viewer gets your permission via OAuth to create a GitHub
+[secret gist](https://help.github.com/articles/about-gists/#secret-gists) and
+saves the report there. Since it's done as your Gist, you maintain full control
+over the sharing of the report and you can delete it at any time. You can revoke
+the Viewer's permission to create gists under your [GitHub settings](https://github.com/settings/applications).
+
+## Performance Experiment
+
+The first version of the [Performance Experiment](https://github.com/GoogleChrome/lighthouse/issues/1143)
+project has landed in 1.5.0. This lets you experiment with your page load performance,
+interactively testing the effects of blocking or delaying assets in your critical
+path **during development**.
+
+When Lighthouse is run with the `--interactive` flag, a special report is generated
+that allows interactive selection of costly page resources. The experiment server
+then reruns Lighthouse on that page with those resources blocked.
+
+![Toggling runtime settings](../../images/2017/02/lighthouse/image_10.png)
+
+## New Documentation
+
+Last but not least, we've modernized the documentation at
+[https://developers.google.com/web/tools/lighthouse/](https://developers.google.com/web/tools/lighthouse/)
+and added new audit references.
+
+![New documentation](../../images/2017/02/lighthouse/image_11.png)
+
+That's it for now!
+
+For all the details on the latest in Lighthouse, see the
+[full release notes](https://github.com/GoogleChrome/lighthouse/releases/tag/1.5.0)
+over on Github. As always, [hit us up](https://github.com/GoogleChrome/lighthouse/graphs/contributors)
+to [report bugs](https://github.com/GoogleChrome/lighthouse/issues), file feature requests,
+or brainstorm [ideas](https://github.com/GoogleChrome/lighthouse/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+bug%22) on what you'd like to see next.
+
```

Comment:
Totally up to you: I might suggest giving a call-out to folks on the team that worked on the release for good feels (e.g 'With thanks to Brendan, Paul, Patrick etc... who helped make this release awesome').